src/docs/sdk/hub_and_scope_refactoring.mdx

```diff
@@ -164,6 +164,16 @@ This image illustrates the behavior of these new APIs and how scope data is appl
 
 For a zoomable version visit the <a href="https://miro.com/app/board/uXjVNtPiOfI=/?share_link_id=140058397661" target="_blank">Miro Board</a>
 
+Here's how the new APIs roughly map to the old-style APIs. In case of `configure_scope` and `push_scope` it's not clear-cut whether to use the current or the isolation scope. The choice essentially depends on how long/for what context the changed/forked scope should be active.
+
+| Old API                              | New API                               | Note                                                                                                     |
+| ------------------------------------ | ------------------------------------- | -------------------------------------------------------------------------------------------------------- |
+| `hub = Hub(Hub.current)` (hub clone) | `with isolation_scope() as scope`     |                                                                                                          |
+| `with configure_scope() as scope`    | `scope = Scope.get_isolation_scope()` | Should affect the whole transaction (one request/response cycle, one execution of a task, etc.).         |
+| `with configure_scope() as scope`    | `scope = Scope.get_current_scope()`   | Should affect the whole span or a part of code isolated via a forked current scope with `new_scope`.     |
+| `with push_scope() as scope`         | `with isolation_scope() as scope`     | We're starting a new transaction-worthy unit (request/response cycle, etc.).                             |
+| `with push_scope() as scope`         | `with new_scope() as scope`           | Should affect a localized part of code, e.g. if the user wants to add specific data to a specific event. |
```

Comment:
I find it confusing that we have the same items listed multiple times for the old API. Perhaps there is a way to make the table so that the items for which there are multiple new API variants are listed only once, but those items span the two rows of the table where the corresponding new API is designed? Basically something like the following:

<img width="387" alt="image" src="https://github.com/getsentry/develop/assets/7881302/676656cc-6cf6-4eaa-8fa2-9153482babcb">

Reply:
This would be great! Lemme see how to do that.

Reply:
Hmm, can't figure out how to do it. The best I could do is this, but not sure if that's better:
![Screenshot 2024-03-22 at 13 51 43](https://github.com/getsentry/develop/assets/131587164/fec067b0-f937-4c73-bb05-f6685d2d8866)

Reply:
Nice, didn't realize we can just use plain HTML. Looking good! I've swapped the backticks for `<pre>`s now, as well.

Reply:
Nice! I also added another commit to vertically center the text in the merged cells

Reply:
Cool! I think we want `<code>` instead of `<pre>`, changing that...

Reply:
![Screenshot 2024-03-22 at 14 31 50](https://github.com/getsentry/develop/assets/131587164/903b2056-77f8-421a-9558-11d85710c6a8)

Looking good!